ci/jenkins/Jenkinsfile.j2

```diff
@@ -51,16 +51,16 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 {% import 'ci/jenkins/macros.j2' as m with context -%}
 
 // NOTE: these lines are scanned by docker/dev_common.sh. Please update the regex as needed. -->
-ci_lint = 'tlcpack/ci-lint:20221025-182121-e41d0ed6e'
-ci_gpu = 'tlcpack/ci-gpu:20221025-182121-e41d0ed6e'
-ci_cpu = 'tlcpack/ci-cpu:20221025-182121-e41d0ed6e'
-ci_minimal = 'tlcpack/ci-minimal:20221025-182121-e41d0ed6e'
-ci_wasm = 'tlcpack/ci-wasm:20221025-182121-e41d0ed6e'
-ci_i386 = 'tlcpack/ci-i386:20221025-182121-e41d0ed6e'
-ci_cortexm = 'tlcpack/ci-cortexm:20221025-182121-e41d0ed6e'
-ci_arm = 'tlcpack/ci-arm:20221025-182121-e41d0ed6e'
+ci_lint = 'tlcpack/ci-lint:20221128-070141-ae4fd7df7'
+ci_gpu = 'tlcpack/ci-gpu:20221128-070141-ae4fd7df7'
+ci_cpu = 'tlcpack/ci-cpu:20221128-070141-ae4fd7df7'
+ci_minimal = 'tlcpack/ci-minimal:20221128-070141-ae4fd7df7'
+ci_wasm = 'tlcpack/ci-wasm:20221128-070141-ae4fd7df7'
+ci_i386 = 'tlcpack/ci-i386:20221128-070141-ae4fd7df7'
+ci_cortexm = 'tlcpack/ci-cortexm:20221128-070141-ae4fd7df7'
+ci_arm = 'tlcpack/ci-arm:20221128-070141-ae4fd7df7'
 ci_hexagon = 'tlcpack/ci-hexagon:20221025-182121-e41d0ed6e'
```

Comment:
Currently times out with the new tags: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/nightly-docker-update/148/pipeline/470